### DISCUSSION TITLE
Too many meshes for smokeview?

### DISCUSSION THREAD

**Discussion by mjerebets** [firemodels/fds#13960] [Q&A] [answered]
Hi, 

I'm hoping to get some advice regarding viewing my simulations in Smokeview. I have 504 meshes (split over 7 nodes) for a simulation that runs for 100s, and I suspect that this might be too large to view in Smokeview. However, I would love to visually validate my results. Are there any good ways to do this? The 13 MB .smv file will open (taking ~10 minutes), I can see my ignition pattern as well as my devices, but when I try to right click there is no menu which pops up. All the requisite files are in the same folder. Would love any advice! 

Thank you!
Marta

**Comment by mcgratta** [comment]
There are no limits except for memory. If you are using Windows, open the Task Manager and monitor the memory (RAM) usage. Hit "o" on the keyboard to see the meshes.

**Reply by mjerebets** [reply]
Thanks, Kevin, I appreciate the response. All meshes look correct. I'm running nowhere near RAM capacity, however... (~30%)

Is there anything I can add to the command line prompt to load the smoke/HRRPUV on startup? Or to make the menu open? Thanks again for your help here.

**Comment by gforney** [comment]
there is an autoload option in smokeview where you can specify files to be
loaded at startup.  I just checked, it is not working.  I started an issue
https://github.com/firemodels/smv/issues/2137

I'll take a look.

On Mon, Dec 30, 2024 at 8:36 PM Marta Jerebets ***@***.***>
wrote:

> Thanks, Kevin, I appreciate the response. All meshes look correct. I'm
> running nowhere near RAM capacity, however... (~30%)
>
> Is there anything I can add to the command line prompt to load the
> smoke/HRRPUV on startup? Or to make the menu open? Thanks again for your
> help here.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/firemodels/fds/discussions/13960#discussioncomment-11701316>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AC6UCRXFQAU6Q77ALKE5A2L2IHYKXAVCNFSM6AAAAABUMR6UJ2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNZQGEZTCNQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Glenn Forney

**Comment by gforney** [comment, marked as answer]
install a test smokeview located here https://github.com/gforney/test_bundles/releases/tag/SMOKEVIEW_TEST
or update your smv repo and build smokeview

use -load_soot and/or -load_hrrpuv on the command line to load these quantities when smokeview first starts up

**Reply by mjerebets** [reply]
Thank you for working on this, Glenn. It works well for me now. I appreciate it.

**Comment by gforney** [comment]
great.  report back if you have any other issues
